### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by 0xsirsaif**

### CODE REVIEW

File: docs/api/asyncio_client.rst [hunk 1]
```diff
@@ -557,8 +557,8 @@ transaction), so we have to redo all the work done.
 Generally it's recommended to not execute any long running
 code within the transaction unless absolutely necessary.
 
-Transactions allocate expensive server resources and having
-too many concurrently running long-running transactions will
+Transactions allocate expensive server resources, and having
+too many concurrently long-running transactions will
```
Comment (raddevon) [inline docs/api/asyncio_client.rst:561]: This looks good with one minor tweak to make it sound a bit more natural:

```suggestion
too many concurrent long-running transactions will
```

Comment (0xsirsaif) [reply docs/api/asyncio_client.rst]: Yes it's. Thank you

**Comment by raddevon** [comment]
Looks great! If you are able to sign our CLA, I will get this merged for you.

**Comment by 0xsirsaif** [comment]
@raddevon Yes, how am I supposed to do this? Are there particular instructions on how to sign?

**Comment by raddevon** [comment]
@0xsirsaif If you scroll down the list of checks at the bottom of the pull request, you should see the CLA check. Click the "Details" link to see the CLA. Scroll to the bottom of that to "sign" by logging in with the same GitHub account you used to submit the PR.

Here's a screenshot:
CleanShot 2023-01-30 at 08 39 49@2x

**Comment by 0xsirsaif** [comment]
@raddevon  I just signed the cla, thank you for your response.

**Review by raddevon — APPROVED** [review]

**Comment by raddevon** [comment]
@0xsirsaif Awesome! Thank you for contributing!

**Pull request merged**